Hearthstone Deck Tracker/Stats/GameStats.cs

```diff
@@ -288,7 +288,11 @@ public int OpponentLegendRank
 
 		public int BrawlLosses { get; set; }
 
+		public string? PlayerHeroCardId { get; set; }
+		public string[]? PlayerHeroClasses { get; set; }
+
 		public string? OpponentHeroCardId { get; set; }
+		public string[]? OpponentHeroClasses { get; set; }
```

Comment:
Add `ShouldSerialize` entries for these so we don't write a bunch of nulls to the game stats files. See https://github.com/HearthSim/Hearthstone-Deck-Tracker/blob/47d565774f7b03bf0482fa9a71b9d09695a96189/Hearthstone%20Deck%20Tracker/Stats/GameStats.cs#L563